_templates/package/new/.eslintrc.ejs.t

```diff
@@ -0,0 +1,15 @@
+---
+to: packages/<%= name %>/.eslintrc
+---
+{
+	"extends": ["@rocket.chat/eslint-config"],
+	"overrides": [
```

Comment:
are these overrides always needed?

Reply:
if every package will have this override, doesn't make sense to make it default, so moving it to the eslint package?